config/v1/types.go

```diff
@@ -5,6 +5,84 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageConfig holds cluster-wide information about how to handle images.  The canonical name is `cluster`
+type ImageConfig struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Format is the format of the name to be built for the system component
+	Format string `json:"format"`
+	// Latest determines if the latest tag will be pulled from the registry
+	Latest bool `json:"latest"`
+
+	// MaxImagesBulkImportedPerRepository controls the number of images that are imported when a user
+	// does a bulk import of a Docker repository. This number defaults to 50 to prevent users from
+	// importing large numbers of images accidentally. Set -1 for no limit.
+	MaxImagesBulkImportedPerRepository int32 `json:"maxImagesBulkImportedPerRepository"`
+	// DisableScheduledImport allows scheduled background import of images to be disabled.
+	DisableScheduledImport bool `json:"disableScheduledImport"`
```

Comment:
I think this is an example of something below the line.  I think this only affects whether a particular controller runs. That seems more suited to a per-binary choice.